sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/config/DashboardConfig.java

```diff
@@ -0,0 +1,71 @@
+package com.alibaba.csp.sentinel.transport.config;
+
+import java.net.InetSocketAddress;
+
+/**
+ * For Dashboard Config
+ *
+ * @author jz0630
+ */
+public class DashboardConfig {
```

Comment:
It might be better to be `HeartbeatConfigEntity` or something else to be more clear, as there will be another `DashboardConfig` class in `sentinel-dashboard`.

Reply:
`HeartbeatConfigEntity` is much better, I have changed it in the new commit.